### PR TITLE
CPM-742: Add a short description for domain objects

### PR DESCRIPTION
**Pull request by antoinearrive**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: components/identifier-generator/back/src/Domain/Model/Delimiter.php [hunk 1]
```diff
@@ -5,6 +5,7 @@
 namespace Akeneo\Pim\Automation\IdentifierGenerator\Domain\Model;
 
 /**
+ * String between each element of the identifier
```
Comment (cemorin) [inline components/identifier-generator/back/src/Domain/Model/Delimiter.php:8]:
```suggestion
 * String between each Properties chosen to create the Structure of the identifier
```
That could explain a bit better the link between the classes?
Or maybe juste replace `element` by `propety` to keep the consistency

File: components/identifier-generator/back/src/Domain/Model/Property/FreeText.php [hunk 2]
```diff
@@ -7,6 +7,7 @@
 use Webmozart\Assert\Assert;
 
 /**
+ * Property for adding free text
```
Comment (cemorin) [inline components/identifier-generator/back/src/Domain/Model/Property/FreeText.php:10]:
```suggestion
 * Property to add a free text
```
And should we add `to the structure or the identifier` for each Property description? (to explain even better to what it is added)

File: components/identifier-generator/back/src/Domain/Model/Target.php [hunk 3]
```diff
@@ -7,6 +7,7 @@
 use Webmozart\Assert\Assert;
 
 /**
+ * Identifier generation target
```
Comment (cemorin) [inline components/identifier-generator/back/src/Domain/Model/Target.php:10]:
```suggestion
 * The attribute code of type identifier that will be impacted by the generation
```

File: components/identifier-generator/back/src/Domain/Model/LabelCollection.php [hunk 4]
```diff
@@ -7,6 +7,7 @@
 use Webmozart\Assert\Assert;
 
 /**
+ * Identifier label by locale
```
Comment (cemorin) [inline components/identifier-generator/back/src/Domain/Model/LabelCollection.php:10]: The collection of the generator labels by locale

File: components/identifier-generator/back/src/Domain/Model/IdentifierGeneratorCode.php [hunk 5]
```diff
@@ -7,6 +7,7 @@
 use Webmozart\Assert\Assert;
 
 /**
+ * Identifier code structure
```
Comment (cemorin) [inline components/identifier-generator/back/src/Domain/Model/IdentifierGeneratorCode.php:10]:
```suggestion
 * The code of the generator
```

Comment (antoinearrive) [reply components/identifier-generator/back/src/Domain/Model/IdentifierGeneratorCode.php]: i replace by "Unique identifier code" what do you think ?

**Review by cemorin — APPROVED** [review]

**Pull request merged**